gbpservice/nfp/bin/nfp_configurator.ini

```diff
@@ -17,6 +17,9 @@ modules_dir=gbpservice.nfp.configurator.modules
 reportstate_interval=10
 periodic_interval=9
 
+visibility_vm_ip_address=
+visibility_port=514
+log_level=info
```

Comment:
Changed to debug

Regards,
Deepak

On Thu, Apr 28, 2016 at 11:44 AM, Mahesh Tambe <notifications@github.com>
wrote:

> In gbpservice/nfp/bin/nfp_configurator.ini
> <https://github.com/oneconvergence/group-based-policy/pull/260#discussion_r61377607>
> :
>
> > @@ -17,6 +17,9 @@ modules_dir=gbpservice.nfp.configurator.modules
> >  reportstate_interval=10
> >  periodic_interval=9
> >
> > +visibility_vm_ip_address=
> > +visibility_port=514
> > +log_level=info
>
> can we set log_level by default to debug as vyos firewall logs doesn't
> come in syslog unless log level is debug.? and this log level will be used
> by vyos only. haproxy doesn't use this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/260/files/f3ba221a8153906b60ecff3b7d8f305a0e841889#r61377607>
>